packaging_automation/requirements.txt

```diff
@@ -1,5 +1,5 @@
 #
-# This file is autogenerated by pip-compile with python 3.8
+# This file is autogenerated by pip-compile
```

Comment:
I think we have different versions of `pip-compile` installed. The autogenerated comments differ on our machines. This is not an important issue I guess, as long as pip-compile does not make behaviour changes. If that happens, we may need to fix our pip-compile versions as well.